src/utils/scroll-lock.js

```diff
@@ -160,7 +160,11 @@ export default {
       document.body.style.removeProperty('width');
 
       // restore scrolled Y position after resetting the position property
-      window.scrollTo(0, Math.abs(scrolledClientY));
+      window.scrollTo({
+        top: Math.abs(scrolledClientY),
+        left: 0,
+        behavior: 'smooth',
```

Comment:
Scroll lock should be instant, this is more of a "hack" to return the scrollbar back to where it was, before locking the scroll.